core/controllers/base.py

```diff
@@ -234,6 +228,14 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             if self.user_id is None else user_settings.role)
         self.user = user_services.get_user_actions_info(self.user_id)
 
+        if (feconf.ENABLE_MAINTENANCE_MODE
```

Comment:
@brianrodri Just to check -- any issues with moving this from where it was previously to here?

Reply:
Nope, this seems good to me.

Reply:
@DubeySandeep Please fix this per @brianrodri's suggestion; maintainability is important. You can set up a group discussion for the name if you need.

Reply:
Yeah, I'm working on it in a sperate thread where @vojtechjelinek has asked for the same! (I'll work with him and get it fixed.)

Reply:
Done!